src/module/actor/sheets/sf-charactersheet.ts

```diff
@@ -27,19 +26,8 @@ export class StarforgedCharacterSheet extends VueActorSheet {
     return this
   }
 
-  close(...args) {
-    this.actor.moveSheet?.close(...args)
-    return super.close(...args)
-  }
```

Comment:
So moving this particular functionality to the parent class is great on its own, but now the opening and closing of the move sheet are in different files. How do you feel about the "magic"-ness of that?

Reply:
yeah, i'm not wild about it. i did consider writing a single PC sheet superclass for both to hold both the shared open/close, then making the SF sheet and the IS sheet extensions thereof. but ultimately i decided i wanted to keep this PR simple. it's something i'd like to add very soon, though.